src/user/views/organization_view.py

```diff
@@ -124,7 +131,10 @@ def _create_image_file(self, data, organization, user):
     def update(self, request, *args, **kwargs):
         user = request.user
         partial = kwargs.pop("partial", False)
-        organization = self.get_object()
+        try:
+            organization = self.get_object()
+        except Exception:
+            return Response({"data": "No permission to get organization"}, status=403)
```

Comment:
I am wondering if we can assume an access issue for any type of exception?

Reply:
I think so

Reply:
`get_object` returns an Http404 error if not found so I will just check for that.

Reply:
The approach can mask legitimate _not found_ cases, i.e., where a non-existing org is requested. I believe a HTTP 404 response for both cases should be fine.